src/main/java/org/objectionary/ObjectsBox.java

```diff
@@ -74,13 +77,56 @@ public Map<String, Map<String, Entity>> content() {
     /**
      * Converts the box of objects to a string.
      * @checkstyle NoJavadocForOverriddenMethodsCheck (10 lines)
-     * @todo #25:30min We have to implement the toString() method.
-     *  It should return a string representation of the objects box,
-     *  which will be used in phi emulator.
      */
     @Override
     public String toString() {
-        return null;
+        if (!this.box.containsKey("ν0")) {
+            throw new IllegalArgumentException("The box does not contain the object ν0.");
+        }
+        final List<String> results = new ArrayList<>(this.box.size());
+        results.add(ObjectsBox.objectToString("ν0", this.box.get("ν0")));
+        for (final Map.Entry<String, Map<String, Entity>> entry : this.box.entrySet()) {
+            if (entry.getKey().equals("ν0")) {
+                continue;
+            }
+            results.add(
+                ObjectsBox.objectToString(entry.getKey(), entry.getValue())
+            );
+        }
+        return String.join("\n", results);
+    }
+
+    /**
+     * Converts an object to a string.
+     * @param name The name of the object.
+     * @param bindings The bindings of the object.
+     * @return The string representation of the object.
+     */
+    private static String objectToString(
```

Comment:
@EugeneDar This method name is compound. Maybe we can change it to `converts` or something similar?

Reply:
@Graur changed it, please have a look again